janusgraph-solr/pom.xml

```diff
@@ -56,19 +54,27 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
             <version>${solr.version}</version>
+            <exclusions>
+                <!-- Use jcl-over-slf4j from gremlin-core -->
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.solr</groupId>
```

Comment:
@wojciechwojcik Sorry from the comment placement I thought you were talking about solr-solrj but I see in #310 you're talking about solr-core, which you're right isn't used directly so can be removed. Thanks!